Tests/SwiftFrameTests/ImageLoaderTests.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import XCTest
+@testable import SwiftFrameCore
+
+class ImageLoaderTests: XCTestCase {
+
+    func testLoadImage() throws {
+        let rep = makeImageRepresentationWithSize(.square100Pixels)
+        guard let cgImage = rep.cgImage else {
+            throw NSError(description: "Could not make CGImage from Bitmap")
+        }
+
+        try ImageWriter.write(cgImage, to: "testing/", locale: "en", deviceID: "testing_device", format: .png)
+        XCTAssertNoThrow(try ImageLoader().loadImage(atPath: "testing/en/en-testing_device.png"))
+
+        try TestingUtility.clearTestingDirectory()
+    }
+
+}
+
+extension CGSize {
```

Comment:
Nit: Might be worth to group this and similar extensions in a separate utility file